source/common/grpc/async_client_impl.h

```diff
@@ -158,12 +158,13 @@ class AsyncStreamImpl : public AsyncStream<RequestType>,
       streamError(Status::GrpcStatus::Internal);
       return;
     }
+    std::string grpc_message = Common::getGrpcMessage(*trailers);
     if (grpc_status.value() != Status::GrpcStatus::Ok) {
-      streamError(grpc_status.value());
+      streamError(grpc_status.value(), std::move(grpc_message));
```

Comment:
Done. My originally thought was to move std::string&& around, but that looked too much.